client/js/app/actions/ExplorerActions.js

```diff
@@ -3,10 +3,10 @@ var AppDispatcher = require('../dispatcher/AppDispatcher');
 var ExplorerConstants = require('../constants/ExplorerConstants');
 var ExplorerStore = require('../stores/ExplorerStore');
 var ExplorerValidations = require('../validations/ExplorerValidations');
-var ValidationUtils = require('../utils/ValidationUtils');
-var ExplorerUtils = require('../utils/ExplorerUtils');
 var NoticeActions = require('./NoticeActions');
 var AppStateActions = require('./AppStateActions');
+var RunValidations = require('../utils/RunValidations');
```

Comment:
We call it with `.run` below, so we're good. We have to do it this way for our tests to pass :/ Stubbing doesn't work with `require` calls that just return a function, as it would require stubbing out portions of the module loader which is craziness.